src/main/java/com/mixpanel/android/surveys/AutoResizeTextView.java

```diff
@@ -0,0 +1,59 @@
+package com.mixpanel.android.surveys;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build;
+import android.text.Layout;
+import android.util.AttributeSet;
+import android.util.TypedValue;
+import android.widget.TextView;
+
+
+public class AutoResizeTextView extends TextView {
+    private int mTextSizeSp = 18;
```

Comment:
can we make these two variables `static final`? I don't think we ever change it.